scripts.py

```diff
@@ -267,7 +267,7 @@ def do_run(self):
         works = True
         offset = 0
         while works:
-            works = self.query.offset(offset).limit(self.batch_size)
+            works = self.query.offset(offset).limit(self.batch_size).all()
```

Comment:
This is an unrelated fix which stops this code from going into an infinite loop. Previously, `if works` always succeeded because `works` was always a `Query` object. This makes `works` into a list, which will eventually be empty.